New glossary terms

noahmanger opened the pull request:
Adds 67 new glossary terms.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  {
+    "term": "Affiliated committees",
+    "definition": "Committees and organizations that are considered one committee for purposes of the contribution limits. 11 CFR 110.3(a)(1). Affiliated committees include (1) All committees established or authorized by a candidate as part of his or her campaign for federal or nonfederal office; and (2) All committees established, financed, maintained or controlled by the same person, group or o ...
+  },
```
Should we be linking these citations?

jenniferthibault replied:
Also: style numbered list as ordered list

dorothyyeager replied:
No need to link citations to regs. (The page the glossary term is on will most likely have a link to the reg.)

jenniferthibault replied:
That's one use case: The other use case is that someone will look up the term in the glossary without it being linked to on the page, in which case they don't have a link to the citation. In either case, it feels like we have the info on the link available, and hosted on the site, so it makes sense to me to cross-link them. I'm ok to leave this for now since it would be a big amount of work, that wasn't scoped for this PR, but will make a follow up issue for us to consider

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Clearly identified candidate",
+    "definition": "A candidate is clearly identified when his or her name, nickname, photograph or drawing appears, or when his or her identity is otherwise apparent through an unambiguous reference such as “the President,” “your Congressman,” or “the incumbent,” or through an unambiguous reference to his or her status as a candidate such as “the Democratic presidential nominee” or “the Republica ...
```
Just want to confirm that the quotation marks in the definition won't conflict with setting the glossary variables?

jenniferthibault replied:
Above in line [136](https://github.com/18F/fec-style/pull/756/files#diff-d35b76e5ae7f7ed6864b60d979ad72e7R136) it looks like the quotation marks were escaped (I think that's the term) with a `/`

noahmanger replied:
Eagle eye! So, typically yes. But it seems this is ok because these are smart quotes, and so they're not conflicting with the quotes that wrap the item.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Connected organization",
+    "definition": "An organization that uses its treasury funds to establish, administer or solicit contributions to a separate segregated fund.  11 CFR 100.6(a)."
```
There is a duplicate space in `fund.  11 CFR 100.6(a).` that should be deleted

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Coordinated communication",
+    "definition": "A communication that satisfies a three-pronged test: <ul class='list--numbered'><li>The communication must be paid for by a person other than a federal candidate, authorized committee, or a political party committee, or any agents of the aforementioned entities with whom the communication is coordinated.</li><li>One or more of the five content standards set forth in 11 CFR 109. ...
```
Do list items need to be separated onto their own line? Or does that markup happen in-line like this?

noahmanger replied:
The markup handles it all.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+    "definition": "A group organized for the purpose of influencing the selection of one or more delegates. The term includes a group of delegates, a group of individuals seeking to become delegates and a group of individuals supporting delegates. 11 CFR 110.14(b)(2)."
+  },
+  {
+    "term": "Direct",
```
Should this term be "to direct" ? even if we attach it to `direct` in context when it's used?

dorothyyeager replied:
Technically, yes, it should be "to direct." - Guide needs that change as well.

dorothyyeager replied:
I think so.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Election cycle",
+    "definition": "The period beginning the day after the previous general election for a given federal office and ending on the date of the general election for that office. The number of years in an election cycle differs according to the federal office sought. The election cycle spans two years for House candidates; four years for presidential candidates; and six years for Senate candidates. S ...
```
The other citations are just listed at the end, not set off with a `See`

dorothyyeager replied:
I would leave the citations as is. The "see" is usually going to go with regulations that are a little open to interpretation, where the ones without it are more slam dunk. So if we are using "see", chances are it was recommended by our lawyers.

dorothyyeager replied:
That's ok.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Express advocacy",
+    "definition": "Unambiguously advocating the election or defeat of a clearly identified federal candidate. There are two ways that a communication can be defined as express advocacy (candidate advocacy): by use of certain “explicit words of advocacy of election or defeat” and by the “only reasonable interpretation” test. See 11 CFR 100.22."
```
Remove `See...` citations?

dorothyyeager replied:
No. - Legal reasons.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Federal Election Activity (FEA)",
+    "definition": "Activity by state, district and local party committees, which may be paid for with federal or – in the case of the first two types – a combination of federal and Levin funds. The four types of federal election activity are as follows: 1) Voter registration activity during the period 120 days before a primary or general election and ending on election day itself; 2) Voter identi ...
```
Style numbered list as an ordered list

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Foreign national",
+    "definition": "(1) An individual who is not a citizen of the United States or a national of the United States and has not been lawfully admitted to the U.S. for permanent residence, as defined in 8 U.S.C. §1101(a)(20); or (2) a foreign principal, as defined in 22 U.S.C. §611(b). 11 CFR 110.20(a)(3)."
```
Stylize numbered list as an ordered list

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Get-Out-The-Vote (GOTV)",
+    "definition": "In regard to FEA, GOTV activity encompasses all means of assisting, encouraging or urging potential voters to vote. This activity includes, but is not limited to: encouraging or urging potential voters to vote, whether by mail (including direct mail), email, in person, by telephone (including prerecorded telephone calls, phone banks and messaging such as SMS and MMS), or by any ...
```
Stylize as a bulleted list?

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Lobbyist/registrant",
+    "definition": "A person who is a current registrant under the Lobbying Disclosure Act, or an individual who is named on a current registration or report filed under the Lobbying Disclosure Act. See 11 CFR 104.22."
```
Remove `See...` before citations?

dorothyyeager replied:
No (see above).

dorothyyeager replied:
No. Legal reasons.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Treasury funds",
+    "definition": "Funds of a corporation or labor organization that are derived from commercial activities or dues payments. Treasury funds may be used for the establishment, administrative and fundraising costs of the organization’s separate segregated fund, as well as for making independent expenditures and contributing to Super PACs and the non-contribution accounts of Hybrid PACs. See 11 CFR ...
```
Remove `See...` before citations?

dorothyyeager replied:
No (see above).

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Bundled contribution",
+    "definition": "A contribution forwarded to a reporting committee by a lobbyist/registrant or lobbyist/registrant PAC, or received by a reporting committee and credited to a lobbyist/registrant or lobbyist/registrant PAC. See 11 CFR 104.22(a)(6)."
```
Remove `See...` before citations?

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
-   {
+  {
     "term": "Separate segregated fund (SSF)",
     "definition": "A political committee established, administered or financially supported by a corporation or labor organization, popularly called a Corporate or Labor Political Action Committee (PAC). See 114.1(a)(2)(iii). The term \"financially supported\" does not include contributions to the SSF, but does include the payment of establishment, administration or solicitation costs. 100.6(b)."
```
Remove `See...` before citations? Most of the following changes don't include it before the list starts

dorothyyeager replied:
No. "See" is usually there for legal reasons.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Member",
+    "definition": "With respect to a labor organization, a trade association, a cooperative or other incorporated membership organization, a member is an individual or other entity that: Satisfies the requirements for membership in a membership organization; affirmatively accepts the organization’s invitation to become a member; and maintains a long-term and continuous bond with the organization  ...
```
Should we make this a bulleted list?

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Solicitation (SSF)",
+    "definition": "A statement that publicizes the SSF’s right to accept unsolicited contributions from any lawful contributor; provides information on how to contribute to the SSF; or encourages support for the SSF. AO 1984–55, n. 2; AOs 1979–66 and 1979–13."
```
It feels like if we're citing AOs we should definitely link to them

dorothyyeager replied:
Agree.

dorothyyeager replied:
Agree.

jenniferthibault replied:
Links:
https://www.fec.gov/data/legal/advisory-opinions/1984-55/
https://www.fec.gov/data/legal/advisory-opinions/1979-66/
https://www.fec.gov/data/legal/advisory-opinions/1979-13/

dorothyyeager replied:
Links to these AOs added.

jenniferthibault commented on js/terms.json:
```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Voter registration activity",
+    "definition": "In regard to FEA, voter registration activity encompasses all means of contacting potential voters to assist, encourage or urge them to register to vote. This activity includes, but is not limited to: Encouraging or urging potential voters to register to vote, whether by mail (including direct mail), email, in person, by telephone (including pre-recorded telephone calls, phone  ...
```
Should we format this as a bulleted list?

jenniferthibault commented:
Thanks so much for following up on my comments @dorothyyeager !

**Next steps:**
- Leave citations as they are
- Correct miscellaneous errors pointed out in comments
- Reformat style for numbered lists as <ol>
- Reformat style for other lists as <li>

@noahmanger is it safe to assume that you'll make these changes as author of the PR? (I know you and Dorothy paired, so not sure what to expect)

noahmanger commented:
Sorry I missed your comment here @jenniferthibault . @dorothyyeager set up a google sheet that should be the source of truth for these definitions, so I think all the changes should be made there and then we'll just paste in the new code. @dorothyyeager can you make the edits?

dorothyyeager commented:
Will do @noahmanger @jenniferthibault

dorothyyeager commented:
Have made the changes to the google spreadsheet.

noahmanger commented:
Moved the changes to terms.json

I also included a CSS fix so that in the future you don't need to add the `class="list--bulleted` or `class="list--numbered`. Now in glossary definitions, `<ol>` (ordered lists) are automatically styled with numbers and `<ul>` (unordered lists) are automatically styled with bullets.

I also removed the left margin on these lists, because things were getting to cramped. Does this work for you @jenniferthibault ?

![image](https://user-images.githubusercontent.com/1696495/29294314-a1fa8c14-8103-11e7-9d44-b6114e16fdc2.png)

jenniferthibault commented:
Works for me in this scenario. Great improvement, thanks!

dorothyyeager commented:
Thanks @noahmanger

noahmanger commented:
@jenniferthibault can you approve the changes if they're good for you? And @patphongs can you pull this down just to make sure everything works as expected?

patphongs commented:
Just removed a small typo (extra period). Everything looks good to go @jenniferthibault
